Worksheet B - Pseudocode.txt

```diff
@@ -0,0 +1,17 @@
+Import random library
+N = Random INT from 1-10
+
+Word = Random String of N Characters
+
+Guess = input ("What is the secret word, if it has " + N + "letters?")
+
+For N until Guess[N] or until after fourth guess:
```

Comment:
You seem to be re-using `N` for several different purposes here (word length and guess number) -- avoid this. Also single-letter variable names are best avoided in favour of more descriptive names.